client/client.go

```diff
@@ -40,19 +40,14 @@ func init() {
 	)
 }
 
-// ErrRepoNotInitialized is returned when trying to can publish on an uninitialized
+// ErrRepoNotInitialized is returned when trying to can an uninitialized
 // notary repository
 type ErrRepoNotInitialized struct{}
 
-// ErrRepoNotInitialized is returned when trying to can publish on an uninitialized
+// ErrRepoNotInitialized is returned when trying to publish an uninitialized
```

Comment:
Delete this comment. Looks like it was copy pasted from the type def and vet/fmt are clever enough not to complain about missing comments on the `Error()` method.